setup.sh

```diff
@@ -146,7 +148,7 @@ deploy_notifier () {
   gcloud run deploy "${SERVICE_NAME}" \
     --image="${IMAGE_PATH}" \
     --no-allow-unauthenticated \
-    --update-env-vars="CONFIG_PATH=${DESTINATION_CONFIG_PATH}" \
+    --update-env-vars="CONFIG_PATH=${DESTINATION_CONFIG_PATH}","PROJECT_ID=${PROJECT_ID}" \
```

Comment:
I would remove the quotes around the comma so the flag value is just one long string.

Reply:
This is mainly a readability/nitpick comment.

Reply:
Resolved!